scriptmodules/helpers.sh

```diff
@@ -1445,19 +1445,24 @@ function dkmsManager() {
     local mode="$1"
     local module_name="$2"
     local module_ver="$3"
-    local kernel="$(uname -r)"
+    local kernel
+    local kernels=($(for kernel in $(dpkg -S /lib/modules/*/ | cut -d ":" -f2); do basename "$kernel" | grep -v "$(uname -r)"; done))
     local ver
 
+    # make sure to include running kernel if not in chroot
+    if [[ "$__chroot" -eq 0 ]]; then
+        kernels+=("$(uname -r)")
+    fi
+
     case "$mode" in
         install)
             if dkms status | grep -q "^$module_name"; then
                 dkmsManager remove "$module_name" "$module_ver"
             fi
-            if [[ "$__chroot" -eq 1 ]]; then
-                kernel="$(ls -1 /lib/modules | tail -n -1)"
```

Comment:
In case it helps explain why I didn't (re)use this to guess the pending kernel... it doesn't seem to be a reliable method to find the target kernel, in a chroot or otherwise. After re-installing the `raspberrypi-headers` and `raspberrypi-kernel` packages, I tried invoking this on a real rpi3, and it erroneously selected the `v8+` folder.